frontend/src/components/utils/AccountLink.tsx

```diff
@@ -2,11 +2,21 @@ import Link from "next/link";
 
 export interface Props {
   accountId: string;
-  children?: React.ReactNode;
 }
 
-export default ({ accountId, children }: Props) => (
-  <Link href="/accounts/[id]" as={`/accounts/${accountId}`}>
-    <a className="account-link">{children || `@${accountId}`}</a>
-  </Link>
-);
+export default ({ accountId }: Props) => {
+  return (
+    <Link href="/accounts/[id]" as={`/accounts/${accountId}`}>
+      <a className="account-link">{truncateAccountId(accountId)}</a>
+    </Link>
+  );
+};
+
+export function truncateAccountId(
+  accountId: string,
+  lengthThreshold: number = 25
+) {
+  return accountId.length > lengthThreshold
+    ? accountId.slice(0, 8) + "..." + accountId.slice(accountId.length - 7)
```

Comment:
Looking at the result of abbreviation of `b7df2090560a225dc4934aed43db03a6c674c2d4.lockup.near` to `b7df2090...up.near`, I think we should do better than that, e.g. `b7df2…2d4.lockup.near` (also, consider using "ellipsis" character … if it looks better in UI)